tests/experimental/datasources/test_postgres_datasource.py

```diff
@@ -71,7 +76,7 @@ def assert_batch_request(
 
 @pytest.mark.unit
 def test_add_table_asset_with_splitter(create_source):
-    with create_source(lambda: None) as source:
+    with create_source(lambda validate_batch_spec: None) as source:
```

Comment:
If the lambda var isn't being used, should we use `_`? I think this is probably fine too.